ocdaction/ocdaction/templates/registration/register.html

```diff
@@ -34,9 +34,9 @@ <h1>Register</h1>
                                 {% endif %}
                             {% endif %}
 
-                        {% if field.name != "password1" %}
+                        {% if field.name == "password1" or "password2" %}
```

Comment:
I don't think we need this `if field.name == "password1" or "password2"` as there is no help text on the other fields that we are trying to hide.  The `if field.name != "password1"` was initially put there because the help text from django is long:

> Your password can't be too similar to your other personal information.
Your password must contain at least 10 characters.
Your password can't be a commonly used password.
Your password can't be entirely numeric.

Reply:
Ah thanks, I forgot to remove it from when I was trying to figure out how it all works in the first place :)
I've added a comment so it's more clear to which section this helptext refers to.